spec/acceptance/default_parameters_jira_10_spec.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+def on_supported_os
+  (fact('os.family') == 'Debian' and fact('os.release.major') == '22.04') or (fact('os.family') == 'RedHat' and fact('os.release.major') >= '8')
```

Comment:
This excludes debian and future ubuntu versions. is that on purpose? And below you mentioned debian-11, but the test isn't running on that platform.

Reply:
debian11 and ubuntu2204 and RH8/9 are included now. Ubuntu 2004 only has postgresql 12 and therefore is not compatible with JIRA >10